src/main/java/com/minecolonies/coremod/entity/pathfinding/PathNavigate.java

```diff
@@ -253,9 +253,9 @@ public void onUpdateNavigation()
 
             if (pEx.isOnLadder())
             {
-                final Vec3d vec3 = this.getPath().getPosition(this.entity);
+                final Vec3d vec3 = this.getPath().getPosition(this.ourEntity);
 
-                if (vec3.squareDistanceTo(entity.posX, vec3.yCoord, entity.posZ) < 0.1)
+                if (vec3.squareDistanceTo(ourEntity.posX, vec3.y, ourEntity.posZ) < 0.1)
                 {
                     //This way he is less nervous and gets up the ladder
                     double newSpeed = 0.05;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 0.05 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)